packages/typography/src/Heading/Heading.tsx

```diff
@@ -1,9 +1,39 @@
-function Heading() {
-  return (
-    <div>
-      <h1 className="text-red-12">My Heading</h1>
-    </div>
-  );
-}
+import clsx from "clsx";
+import React from "react";
+
+import { alignMap, colorMap, sizeMap } from "../helpers/prop-mappings";
+
+type HeadingProps = React.ComponentProps<"h1"> & {
+  align?: keyof typeof alignMap;
+  size?: keyof typeof sizeMap;
+  color?: keyof typeof colorMap;
+};
+
+type HeadingRef = HTMLHeadingElement;
+
+const Heading = React.forwardRef<HeadingRef, HeadingProps>(
+  (
+    {
+      align = "DEFAULT",
+      color = "DEFAULT",
+      size = "DEFAULT",
+      ...props
+    }: HeadingProps,
+    forwardedRef,
+  ) => {
+    return (
+      <h3
```

Comment:
Feels like an oversight to only have this as an `h3` which then makes me feel like we need an `as` prop which opens up a big-old-can-of-worms.

Reply:
I thought about making the `as` prop a required field here so that the engineer has to decide which heading level to use. But as it exists now there can be an as prop passed optionally. What do you think about making it required?

Reply:
I think making it required is good because it forces you to think about the heading level. Could see everything being an h3 otherwise. I'm an old school semantic HTML guy though so maybe need to take this through that lens

Reply:
I can make it required, I was going to initially but that might be too prescriptive. Will update!